docs/CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ created by v1.90.0 or newer versions. The solution is to upgrade to v1.90.0 or n
 * FEATURE: [vmui](https://docs.victoriametrics.com/#vmui): add `apply` and `cancel` buttons to settings popup. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4013).
 * FEATURE: [vmctl](https://docs.victoriametrics.com/vmctl.html): automatically disable progress bar when TTY isn't available. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3823).
 * FEATURE: [vmauth](https://docs.victoriametrics.com/vmauth.html): add `-configCheckInterval` command-line flag, which can be used for automatic re-reading the `-auth.config` file. See [this feature request](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3990).
+* FEATURE: [vmauth-enterprise](https://docs.victoriametrics.com/vmauth.html): add `ip_filters` section in config file with ip ranges `allow_list` and `deny_list`. See [this feature request](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3491) and [these docs](https://docs.victoriametrics.com/vmauth.html#ip-filters).
```

Comment:
There is no need in backporting doc changes from enterprise pull request to open-source pull request - these changes are usually backported to open-source code via `git cherry-pick` after merging the enterprise pull request in the same way as all the merged pull requests are backported across different branches (`master`, `cluster`, `enterprise*`, `lts*`, etc.)